desktop/storeService.ts

```diff
@@ -17,7 +32,7 @@ class StoreService {
     return StoreService.store.get(key);
   };
 
-  static remove = ({ key }: { key: string }) => {
+  static remove = ({ key }: { key: keyof TypedSchemaStore }) => {
```

Comment:
What about such keys: `accounts.someAccountId`, `accout.someAccountId.someProp`?
I think that will be cool to get hinting for any possible key on any nesting level, but I think we can live without it for a while. So let's revert it to `string`.

Reply:
`accounts.someAccountId` will work with the new implementation